telegram/ext/jobqueue.py

```diff
@@ -103,6 +103,7 @@ def _put(self, job, time_spec=None, previous_t=None):
         # enqueue:
         self.logger.debug('Putting job %s with t=%f', job.name, time_spec)
         self._queue.put((next_t, job))
+        job._next_t = next_t
```

Comment:
Accessing a protected member upsets Codacy. Please add a setter for `Job.next_t` and use that one here.

Reply:
Added a setter. But now setter for `Job.next_t` requires timestamp value and getter returns value in a different format, `datetime`, which is kind of counterintuitive. Is it a problem?

Reply:
Good point. Could you make the setter accept both timestamp and datetime by converting datetime to timestamp using `helpers.to_float_timestamp()`? And raise a `TypeError` if a different type is passed.
The setter won't be used often, but still …

Reply:
Added.